src/js/themes/base.js

```diff
@@ -945,6 +949,70 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    pagination: {
+      // container: {
+      //   // any box props,
+      //   // extend: undefined,
+      // },
+      button: {
+        color: 'text-strong',
+        active: {
+          background: {
+            color: 'active-background',
+          },
+        },
+        hover: {
+          background: {
+            color: 'background-contrast',
+          },
+          color: undefined,
+        },
+      },
+      // control: {
+      //   // extend: undefined,
+      //   size: {
+      //     small: {
+      //       font: { ...fontSizing(-1) },
+      //       height: `${baseSpacing * 1.5}px`,
+      //       width: `${baseSpacing * 1.5}px`,
+      //       border: {
+      //         radius: `${baseSpacing / 6}px`, // 4
+      //         width: '2px',
+      //       },
+      //     },
+      //     medium: {
+      //       font: { ...fontSizing(0) },
+      //       height: `${baseSpacing * 1.5}px`,
+      //       width: `${baseSpacing * 1.5}px`,
+      //       border: {
+      //         radius: `${baseSpacing / 4}px`, // 6
+      //         width: '2px',
+      //       },
+      //     },
+      //     large: {
+      //       font: { ...fontSizing(1) },
+      //       height: `${baseSpacing * 2}px`,
+      //       width: `${baseSpacing * 2}px`,
+      //       border: {
+      //         radius: `${baseSpacing / 2}px`, // 12
+      //         width: '2px',
+      //       },
+      //     },
+      //   },
+      // },
```

Comment:
Why is this block needed? Is it still relevant?

Reply:
If we choose to implement the `size` prop, yes (https://github.com/grommet/grommet/pull/4646#discussion_r561400324). If we choose not to include it in the first pass, then we will remove it.

Reply:
OK, so I'll keep this thread open so we can circle back to it before merging.